nexus-repository-composer/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sonatype.nexus.plugins</groupId>
+    <artifactId>composer-parent</artifactId>
+    <version>0.0.2</version>
```

Comment:
Perhaps create an Issue in this project to track adding a "-SNAPSHOT" suffix to versioning (as was done in another project).

Reply:
Issue created: https://github.com/sonatype-nexus-community/nexus-repository-composer/issues/50